### PR TITLE
Correct sensuctl command formatting in RBAC reference

### LINKED ISSUE #2453: Docs Feedback - sensuctl role create command is incorrect
Link to page: 
https://docs.sensu.io/sensu-go/latest/reference/rbac/#create-roles

### Feedback

`sensuctl role create prod-admin --verb get,list,create,update,delete --resource * --namespace production`

Should be 

`sensuctl role create prod-admin --verb='get,list,create,update,delete' --resource='*' --namespace production`

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Corrects the formatting (adds `=` and single quotes) in sensuctl commands in the RBAC reference

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2453

### CODE REVIEW

**Comment by hillaryfraley** [comment]
@asachs01 I found one other command with a `verb` flag and added the formatting there too.

I noticed that there are many other sensuctl commands in the RBAC reference that do not use `=` and single quotes. For example, should commands like this:
```
sensuctl role-binding create prod-admin-oncall --role=prod-admin --group=oncall
```
be reformatted like this:
```
sensuctl role-binding create prod-admin-oncall --role='prod-admin' --group='oncall'
```

And there are other inconsistencies...like another command uses `--groups` (with an 's') rather than `--group` (singular). And https://docs.sensu.io/sensu-go/latest/reference/rbac/#manage-users does not use `=` but https://docs.sensu.io/sensu-go/latest/reference/rbac/#create-users does. 

Is there a rule about how sensuctl commands for RBAC must be formatted that I could apply throughout this doc?

**Comment by hillaryfraley** [comment]
@asachs01 I found one other command with a `verb` flag and added the formatting there too.

I noticed that there are many other sensuctl commands in the RBAC reference that do not use `=` and single quotes. For example, should commands like this:
```
sensuctl role-binding create prod-admin-oncall --role=prod-admin --group=oncall
```
be reformatted like this:
```
sensuctl role-binding create prod-admin-oncall --role='prod-admin' --group='oncall'
```

And there are other inconsistencies...like another command uses `--groups` (with an 's') rather than `--group` (singular). And https://docs.sensu.io/sensu-go/latest/reference/rbac/#manage-users does not use `=` but https://docs.sensu.io/sensu-go/latest/reference/rbac/#create-users does. 

Is there a rule about how sensuctl commands for RBAC must be formatted that I could apply throughout this doc?

**Comment by asachs01** [comment]
Oof, shoot. I don't know. I feel like we might need @palourde 's input here? I feel like stumbling on this is making me second guess any preconceived ideas I had about sensuctl's grammar.

**Comment by palourde** [comment]
We use this library to parse flags: https://github.com/spf13/pflag/#command-line-flag-syntax

As you can see, there's not a single way of defining the flags. The safest way is probably `--role='prod-admin'` because it will deal with special characters and whitespace. That being said, `--role prod-admin` also works and might be easier to read/edit. 🤷

**Comment by hillaryfraley** [comment]
Thank you, @palourde . Is there any place I can look to settle the plural vs. singular question? For example, the commands under https://docs.sensu.io/sensu-go/latest/reference/rbac/#remove-a-user-from-a-group use both `remove-group` and `remove-groups`. And here we have `add-group` and `set-groups`: https://docs.sensu.io/sensu-go/latest/reference/rbac/#assign-a-user-to-a-group

**Comment by palourde** [comment]
These are all different and valid subcommands:

```
$ sensuctl user -h
Manage users

[...]
Commands:
  add-group       add group to user given username and group
[...]
  remove-group    remove group from user given username and group
  remove-groups   remove all the groups for a given user
  set-groups      set the groups for a given user
[...]
```

Then, for the flag, unfortunately there might be some inconsistencies, depending on the type of flag e.g.

https://github.com/sensu/sensu-go/blob/16392dbff4ee2513ac1bcf7312a10eb862f5f51d/cli/commands/rolebinding/create.go#L105-L107
vs
https://github.com/sensu/sensu-go/blob/16392dbff4ee2513ac1bcf7312a10eb862f5f51d/cli/commands/user/create.go#L75

**Comment by hillaryfraley** [comment]
🤦 thank you @palourde -- I was so busy overthinking it I didn't remember `help`

File: content/sensu-go/5.20/reference/rbac.md [hunk 1]
```diff
@@ -537,7 +537,7 @@ You can use [sensuctl][2] to create a cluster role.
 For example, the following command creates a global event reader role that can read only events across all namespaces within Sensu.
 
 {{< highlight shell >}}
-sensuctl cluster-role create global-event-reader --verb get,list --resource events
+sensuctl cluster-role create global-event-reader --verbs='get,list' --resource='events'
```
Comment (hillaryfraley) [inline content/sensu-go/5.20/reference/rbac.md]:
```suggestion
sensuctl cluster-role create global-event-reader --verb='get,list' --resource='events'
```

**Pull request merged**